dbaas/psmdb/config.go

```diff
@@ -15,6 +15,7 @@
 package psmdb
 
 import (
+	"github.com/Percona-Lab/percona-dbaas-cli/dbaas"
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#imports

I tend to organize imports in three groups
```
stdlib

external dependencies

internal dependencies
```
for example https://github.com/Percona-Lab/percona-dbaas-cli/blob/add-psmdb/dbaas/pxc/pxc.go#L17-L28